Modules/Brand/Http/Services/BrandService.php

```diff
@@ -178,6 +166,40 @@ public function get(int $userId): array
         ];
     }
 
+    /**
+     * Get a listing of the Brands
+     *
+     * @param $requestData
+     * @return array
+     */
+    public function getBrands($requestData): array
+    {
+
+        $user = User::find($requestData->user_id);
+        if ($user) {
+            $brandUsers = User::where('country_id', $user->country_id)->where('role', 'brand')->get();
+        } else {
+            $brandUsers = User::where('role', 'brand')->get();
+        }
+
+        if ($brandUsers) {
+            foreach ($brandUsers as $brandUser) {
+                $brand = Brand::where('user_id', $brandUser['id'])->where('go_live', '2')->first();
+                if ($brand) {
+                    $data[] = array(
+                        'brand_key' => $brand->bazaar_direct_link,
+                        'brand_id' => $brand->id,
+                        'brand_name' => $brand->brand_name,
+                        'brand_logo' => $brand->logo_image != '' ? asset('public') . '/' . $brand->logo_image : asset('public/img/logo-image.png'),
+                    );
+                }
+
+            }
+        }
```

Comment:
Use it like this:
$user = User::find($requestData->user_id);

$brandUsers = User::where('role', 'brand');

if ($user) {
    $brandUsers->where('country_id', $user->country_id);
}

$brandUsers = $brandUsers->get();

$brandUserIds = $brandUsers->pluck('id');

$brands = Brand::whereIn('user_id', $brandUserIds)
    ->where('go_live', '2')
    ->get();

$data = [];

foreach ($brands as $brand) {
    $brandLogo = $brand->logo_image ? asset('public/' . $brand->logo_image) : asset('public/img/logo-image.png');

    $data[] = [
        'brand_key' => $brand->bazaar_direct_link,
        'brand_id' => $brand->id,
        'brand_name' => $brand->brand_name,
        'brand_logo' => $brandLogo,
    ];
}